pyproject.toml

```diff
@@ -27,7 +28,7 @@ dependencies = [
   "requests-cache",
   "rich==13.3.4",
   "multiversx-sdk-network-providers>=0.11.0",
-  "multiversx-sdk-wallet>=0.7.0",
+  "multiversx-sdk-wallet>=0.8.0",
```

Comment:
Here, we should protect against breaking changes, and add upper bounds, as well. Perhaps for all our libraries?

In requirements.txt, as well.

Reply:
Done for our libraries